### ISSUE TITLE
Not work for mac

### ISSUE THREAD

**Issue by tulu1688** [h2non/videoshow#66]
I used videoshow in mac os. The prerequisite ffmpeg app installed by "brew install ffmpeg". But when running videoshow I got the log

```
Error: Error: ffmpeg exited with code 1: Error reinitializing filters!
Failed to inject frame into filter network: Invalid argument
Error while processing the decoded data for stream #14:0
Conversion failed!

    at ChildProcess.<anonymous> (/Users/kai.tran/tmp/videoshow/node_modules/fluent-ffmpeg/lib/processor.js:182:22)
    at ChildProcess.emit (events.js:180:13)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:12)
ffmpeg stderr: undefined

```

**Comment by Mashpy** [comment]
same here on the mac.

**Comment by Mashpy** [comment]
I have found the solution. When I was using videoshow, I had put some different resolution images. For this reason, I got this same error. image resolution should same.

Also I faced this error message when I was trying to compile "node subtitles.js". Then I have found that I had not installed ffmpeg properly. So I uninstalled.

$ brew remove ffmpeg 

Then reinstall like this =>

brew install ffmpeg --with-vpx --with-vorbis --with-libvorbis --with-vpx --with-vorbis --with-theora --with-libogg --with-libvorbis --with-gpl --with-version3 --with-nonfree --with-postproc --with-libaacplus --with-libass --with-libcelt --with-libfaac --with-libfdk-aac --with-libfreetype --with-libmp3lame --with-libopencore-amrnb --with-libopencore-amrwb --with-libopenjpeg --with-openssl --with-libopus --with-libschroedinger --with-libspeex --with-libtheora --with-libvo-aacenc --with-libvorbis --with-libvpx --with-libx264 --with-libxvid

Now run =>
node subtitle.js


It's working !
